app/helpers/application_helper.rb

```diff
@@ -1,7 +1,7 @@
 module ApplicationHelper
   def display_picture(game)
-    if game.picture.nil?
-      image_tag('app/assets/images/fallback/default.png')
+    if game.nil? || game.picture.nil?
```

Comment:
You can replace this line with `if game&.picture.nil?`

http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/